src/Devtools/Stylish.hs

```diff
@@ -0,0 +1,42 @@
+module Devtools.Stylish
+  ( Stylish.Config
+  , getHaskellFiles
+  , getStylishConfig
+  , testTree
+  )
+where
+
+import Devtools.Prelude
+import Test.Tasty.MGolden
+
+import qualified Data.List                as List
+import qualified Data.Text                as Text
+import qualified Data.Text.IO             as Text
+import qualified Language.Haskell.Stylish as Stylish
+import qualified Test.Tasty               as Tasty
+
+testTree :: Stylish.Config ->[FilePath] -> Tasty.TestTree
+testTree config = Tasty.testGroup "Stylish-Haskell" . fmap singleTest
+  where
+    singleTest :: FilePath -> Tasty.TestTree
+    singleTest filePath = goldenTest filePath filePath $ getFormatted config filePath
```

Comment:
Maybe it's too "magical" but this could also be written as:

```haskell
singleTest = join goldenTest `ap` getFormatted config
```

My Haskell skills are not as advanced as yours, so I suspect your existing code is better, but I figured I'd still submit it for consideration just in case.

Reply:
I like this but its quite magical. I am not sure its worth the extra brain cycles it invokes. I feel inclined to leave as is.